go.mod

```diff
@@ -5,15 +5,10 @@ go 1.18
 require (
 	github.com/joho/godotenv v1.5.1
 	github.com/stretchr/testify v1.8.4
-	golang.org/x/oauth2 v0.10.0
```

Comment:
For theses unused packages,I believe this comes from gen_api.sh

Reply:
It might be the case, yeah. But since this is run locally, they got accidentally committed.